installation_and_upgrade/vhd_build.bat

```diff
@@ -0,0 +1,6 @@
+set PYTHONUNBUFFERED=1
+set "SOURCE=\\isis.cclrc.ac.uk\inst$\Kits$\CompGroup\ICP\Releases"
+call "%~dp0\define_latest_genie_python.bat"
+
+call "%LATEST_PYTHON%" "%~dp0IBEX_upgrade.py" --kits_icp_dir "%KITS_ICP_PATH%" create_vhds --quiet
+IF ERRORLEVEL 1 EXIT /b %errorlevel%
```

Comment:
IF %ERRORLEVEL% NEQ 0         is usually recommended instead of      IF ERRORLEVEL 1      as it covers an exit code of -1